dali/interface.py

```diff
@@ -9,68 +7,27 @@ class CommunicationError(Exception):
     pass
 
 
-class DaliServer(object):
-    """Communicate with daliserver
-    (https://github.com/onitake/daliserver)
+class ParameterError(Exception):
+    pass
 
-    NB this requires daliserver commit
-    90e34a0cd2945dc7a15681f11647e708f858521e or later.
+class DriverInterface(object):
```

Comment:
Should the class name either be Driver or Interface? DriverInterface is somewhat confusing, even InterfaceDriver seems more appropriate.